hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/TopologySubcommand.java

```diff
@@ -136,10 +144,11 @@ public Class<?> getParentType() {
   // Format
   // Location: rack1
   //  ipAddress(hostName) OperationalState
-  private void printOrderedByLocation(List<HddsProtos.Node> nodes) {
+  private void printOrderedByLocation(List<HddsProtos.Node> nodes,
+                                      String state) throws IOException {
     HashMap<String, TreeSet<DatanodeDetails>> tree =
```

Comment:
nit: interface over implementation.
```suggestion
    Map<String, TreeSet<DatanodeDetails>> tree =  new HashMap<>();
    Map<DatanodeDetails, HddsProtos.NodeOperationalState> operationalState =
        new HashMap<>();
```

Reply:
Thank you for the suggestions! I have made the changes.